filesystems/xfs/xfstests/README.md

```diff
@@ -0,0 +1,28 @@
+xfstests test suite
+=====================================
+Wrapper for xfstests test suite, see http://git.kernel.org/?p=fs/xfs/xfstests-dev.git;a=summary \
+Test Maintainer: [Xiong Murphy Zhou](mailto:xzhou@redhat.com)
+
+How to run it
+-------------
+Make sure you have at least 25G free disk space in this partition
+```bash
+$ df -H .
+```
+
+Clone the tests-beaker repository
+```bash
+$ git clone https://github.com/CKI-project/tests-beaker.git
+```
+
+Configure which test cases to run, e.g.
+```bash
+$ export TEST_PARAM_FSTYPE=xfs
+$ export TEST_PARAM_RUNTESTS=generic/001
+```
+
+Execute the test
```

Comment:
What's missing, I think, is the list of dependencies, where to get them, and how to install them. This is especially important for outside developers. The best way to produce such instructions would be to get a fresh install of e.g. Fedora and try to make the test work there.

If you'd rather target these instructions at internal developers only for now, then just a quick mention of what needs to be installed would be enough. Might be a good idea to ask an actual developer for a review :)

Lastly, we don't want to repeat all this (and probably some other things) for all the tests. So the common instructions could go into the README.md in the root of the repo, and the test README.md's could refer to the top one for pre-requisite instructions.

Reply:
@jencce can you provide the details here regarding dependencies ? Also yes we can have a developer review the final version. I'm not sure if there will be any common instructions after these changes, other than make run, but we can move them to the root repo if so.

Reply:
My thought is that, xfstests is a community project. We backport it to beaker from its upstream. Now we are trying to introduce it at another community project(if I understand it right -:),  we can just point to xfstests its own README:
https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git/tree/README

There is a dep-install.sh to install all dependencies.

All filesystem relative developers know xfstests, most of them are contributing to xfstests upstream.

Reply:
Sounds good, @jencce, then we can simply refer to the upstream instructions from here, if they're present in our repo. However, if we're going to instruct people to run the Beaker compatibility wrapper, then we gotta explain how to install the dependencies for it.

Reply:
I added more details based on @jencce response, @spbnick can you take another look ? I'm trying to leave beaker out of it completely and only provide simple basic steps anyone internal/external can run.